sycl/include/CL/sycl/queue.hpp

```diff
@@ -308,6 +308,10 @@ class queue {
     });
   }
 
+  /// Returns whether the queue is in order or OoO
+  ///
+  /// Equivalent to has_property<property::queue::in_order>()
+  bool is_in_order() { return impl->is_in_order(); }
```

Comment:
Why not implement the method as has_property<property::queue::in_order>() then?

Reply:
Ok